ui/lib/core/addon/components/json-editor.hbs

```diff
@@ -51,7 +51,7 @@
       mode=@mode
       readOnly=@readOnly
       theme=@theme
-      viewportMarg=@viewportMargin
+      viewportMargin=@viewportMargin
```

Comment:
The default is inherit from CodeMirror. If you don't pass `viewportMargin` it will default to 10, which was it was doing when I incorrectly spelled the param.

Reply:
Oh - is that default just builtin? In codemirror I see a default of an empty string ([here](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/modifiers/code-mirror.js#L79)) Maybe we could include that in the JSDOC below that it defaults to 10?